ui/app/templates/components/namespace-link.hbs

```diff
@@ -5,11 +5,7 @@
     <div class="level is-mobile">
       <span class="level-left">{{this.namespaceDisplay}}</span>
       <span class="level-right">
-        {{! template-lint-configure no-nested-interactive "warn" }}
-        <button type="button" class="button is-ghost icon">
-          <Chevron @isButton={{true}} class="has-text-grey" />
-        </button>
-        {{! template-lint-configure no-nested-interactive "on" }}
+        <Chevron @isButton={{true}} class="button is-ghost icon has-text-grey" />
```

Comment:
Adding the classes directly to the `Chevron` preserves styling and since there was no `onclick` handler on the `button` I was able to remove it.

Reply:
only thoughts about this approach are accessibility. Does Chevron use the button element? (screen readers, from what I understand, use the keyword button to help indicate what can be clicked).